sample/kotlin/src/main/kotlin/com/datadog/android/sample/sessionreplay/ImageScalingFragment.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sample.sessionreplay
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.datadog.android.sample.R
+
+internal class ImageScalingFragment : Fragment() {
+    override fun onCreateView(
```

Comment:
probably you can remove this override and just use constructor with layout: `ImageScalingFragment : Fragment(R.layout.fragment_image_scaling)`

Reply:
yes, good catch. This fragment becomes essentially a one-liner